client/src/extension.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  LanguageClient,
+  LanguageClientOptions,
+  ServerOptions,
+  TransportKind,
+} from 'vscode-languageclient'
+import { ExtensionContext, workspace } from 'vscode'
+import path from 'path'
+
+let client: LanguageClient
+
+export async function activate(context: ExtensionContext) {
+  // The server is implemented in node
+  const serverModule = context.asAbsolutePath(
+    path.join('server', 'out', 'server.js'),
+  )
+  // The debug options for the server
+  // --inspect=6009: runs the server in Node's Inspector mode so VS Code can attach to the server for debugging
+  const debugOptions = { execArgv: ['--nolazy', '--inspect=6009'] }
+
+  // If the extension is launched in debug mode then the debug server options are used
+  // Otherwise the run options are used
+  const serverOptions: ServerOptions = {
+    run: { module: serverModule, transport: TransportKind.ipc },
+    debug: {
+      module: serverModule,
+      transport: TransportKind.ipc,
+      options: debugOptions,
+    },
+  }
+
+  // Options to control the language client
+  const clientOptions: LanguageClientOptions = {
+    // Register the server for plain text documents
+    documentSelector: [{ scheme: 'file', language: 'prisma' }],
+    synchronize: {
+      // Notify the server about file changes to '.clientrc files contained in the workspace
+      fileEvents: workspace.createFileSystemWatcher('**/.clientrc'),
```

Comment:
What's the `.clientrc` file for?

Reply:
I'm not so sure, in the [docs](https://code.visualstudio.com/api/language-extensions/language-server-extension-guide) it was used without mentioning it. From what I understand from [this](https://github.com/microsoft/vscode/issues/2200) issue, this is probably not needed here.

Reply:
Okay. It seems like some local configuration you can set to adjust the extension. I say we remove  it and add it back in if someone requests it.